Enigmatry.Entry.Blueprint.Common.Tests/CommonVerifierSettingsInitializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Runtime.CompilerServices;
+using Argon;
+using Enigmatry.Entry.Blueprint.Infrastructure.Api.Init;
+using Enigmatry.Entry.SmartEnums.VerifyTests;
+
+namespace Enigmatry.Entry.Blueprint.Common.Tests;
```

Comment:
New project (Common.Tests). Idea is to move common test related things from Infrastructure here (e.g. Builders). We keep adding this project later.